manticore/core/workspace.py

```diff
@@ -58,6 +58,28 @@ class Store(object):
 
     Implement either save_value/load_value in subclasses, or save_stream/load_stream, or both.
     """
+    @classmethod
+    def store_type(cls):
+        raise NotImplementedError("Store subclass must define store_type() class method")
+
+    @classmethod
+    def create_store(cls, desc):
```

Comment:
naming here? Maybe `Store.fromdescriptor`? The `.from...` idiom is from the datetime module which is the primary stdlib module I know of that uses classmethods, so maybe we should copy that